pinterest/teletraan
# Host Type Value Update

Background: selectedHostType value was not update properly which was preventing a cluster update

Solution: add additional code to make sure the value is update when the selector is updated

Test:
1. Navigate to a Cluster Configuration and check the selectedHostType value id in the console
2. <img width="1515" alt="Screenshot 2023-02-09 at 2 15 21 PM" src="https://user-images.githubusercontent.com/69278532/217952473-4962b6ae-8c31-4e4d-8e47-fb44e63c6b2a.png">
3. Update the value in the host type selector and confirm that  the selectedHostType value id in the console has updated
4. <img width="1518" alt="Screenshot 2023-02-09 at 2 15 44 PM" src="https://user-images.githubusercontent.com/69278532/217952393-e7e34c4a-0169-4cd2-8d66-4224b7a6ee28.png">
5. Save the cluster configuration 
6. 
<img width="1584" alt="Screenshot 2023-02-09 at 2 16 03 PM" src="https://user-images.githubusercontent.com/69278532/217952359-1da1b796-9a68-41d8-96c5-f8b0253514d2.png">
8. Refresh the page to validate that the cluster configuration was updated 
9. 
<img width="1708" alt="Screenshot 2023-02-09 at 2 16 13 PM" src="https://user-images.githubusercontent.com/69278532/217952298-815f8687-9d38-436b-ab8b-a60d7ec3f9cd.png">
